third_party/WebKit/Source/modules/vr/NavigatorVR.cpp

```diff
@@ -104,7 +104,7 @@ const char* NavigatorVR::supplementName() {
   return "NavigatorVR";
 }
 
-void NavigatorVR::enqueueVREvent(VRDisplayEvent* event) {
+void NavigatorVR::enqueueVREvent(Event* event) {
```

Comment:
Is this still needed? I think we are not using enqueueVREvent anymore. Better leave it as it was.

Reply:
Good point. Will change back.